contract-examples/tasks.ts

```diff
@@ -215,11 +216,57 @@ task("feeManager:get", "gets fee config")
   })
 
 
-// npx hardhat minter:readRole --network local --address [address]
+// npx hardhat feeManager:readRole --network local --address [address]
 task("feeManager:readRole", "a task to get the network deployer minter list")
   .addParam("address", "the address you want to know the minter role for")
   .setAction(async (args, hre) => {
     const allowList = await hre.ethers.getContractAt("IFeeManager", FEE_MANAGER_ADDRESS)
     await getRole(allowList, args.address)
   })
 
+
+// npx hardhat rewardManager:currentRewardAddress --network local
+task("rewardManager:currentRewardAddress", "a task to get the current configured rewarding address")
+  .setAction(async (_, hre) => {
+    const rewardManager = await hre.ethers.getContractAt("IRewardManager", REWARD_MANAGER_ADDDRESS)
+    const areFeeRecipientsAllowed = await rewardManager.areFeeRecipientsAllowed()
+    const result = await rewardManager.currentRewardAddress()
+    if (areFeeRecipientsAllowed) {
+      console.log("Custom Fee Recipients are allowed. (%s)", result)
+    } else {
+      console.log(`Current reward address is ${result}`)
+    }
+  })
+
+// npx hardhat rewardManager:areFeeRecipientsAllowed --network local
+task("rewardManager:areFeeRecipientsAllowed", "a task to get whether the fee recipients are allowed")
```

Comment:
```suggestion
task("rewardManager:areFeeRecipientsAllowed", "a task to get whether custom fee recipients are allowed to receive rewards")
```

Reply:
should we drop the "a task to..." from these since the other descriptions don't have it?